parse/js/src/main/scala/io/circe/parse/package.scala

```diff
@@ -18,7 +19,7 @@ package object parse extends Parser {
       Xor.left(ParsingFailure(error.message, exception))
   }
 
-  private[this] def convertJson(j: Any): Json = j match {
+   def convertJson(j: Any): Json = j match {
```

Comment:
We need the new module in part so that we can keep the public API of `parse` consistent across JVM and JS, which means we'll either need to make this `private[circe]` or to move it to `scalajs` and switch the direction of the dependency.